scripts/.airnotmin.toml

```diff
@@ -0,0 +1,34 @@
+# Using `git update-index --assume-unchanged scripts/.air.toml` to let git untrack it.
```

Comment:
Don't need to commit this file

Reply:
these are original commits...i just remove the minimal tag

Reply:
Also remove this file please

Reply:
ok